package homework;

public class PrinterDemo {
	public static void main(String[] args) {
		Printer p = new Printer03(20, true);
		p.print(25);
		p.setDuplex(false);
		p.print(10);
	}

}
